cmd/release/cmd/push.go

```diff
@@ -37,8 +42,77 @@ var pushK3sTagsCmd = &cobra.Command{
 	},
 }
 
+var pushChartsCmd = &cobra.Command{
+	Use:     "charts [branch-line] [debug (optional)]",
+	Short:   "Push charts updates to remote upstream charts repository",
+	Example: "release push charts 2.9",
+	ValidArgsFunction: func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
+		if err := validateChartConfig(); err != nil {
+			fmt.Println(err)
+			os.Exit(1)
+		}
+
+		if len(args) == 0 {
+			return rootConfig.Charts.BranchLines, cobra.ShellCompDirectiveNoFileComp
+		}
+
+		return nil, cobra.ShellCompDirectiveNoFileComp
+	},
+	Args: func(cmd *cobra.Command, args []string) error {
+		if err := validateChartConfig(); err != nil {
+			fmt.Println(err)
+			os.Exit(1)
+		}
+
+		if len(args) < 1 {
+			return errors.New("expected 1 argument: [branch-line]")
+		}
+
+		var (
+			releaseBranch string
+			found         bool
```

Comment:
This variable doesn't need to be declared or even used below. The function call be be directly in the `if` statement.